docs/source/example_use.rst

```diff
@@ -51,12 +51,31 @@ set up Orchestra yet, you should do that now. Also, make sure that
 have loaded the workflow into the database (``python manage.py loadworkflow
 journalism_workflow v1``).
 
-The journalism workflow requires Google Apps integration to run,
-so make sure in ``orchestra_settings.py`` you set ``settings.GOOGLE_APPS`` to
-``True``, and fill in values for ``settings.GOOGLE_SERVICE_EMAIL``,
-``settings.GOOGLE_P12_PATH``, and ``settings.GOOGLE_PROJECT_ROOT_ID``. Set up
-and correct values for those settings are described in `the Google Apps
-documentation <https://developers.google.com/drive/web/delegation>`_.
+The journalism workflow requires Google Apps integration to run, so
+make sure in ``orchestra_settings.py`` you set
+``settings.GOOGLE_APPS`` to ``True``, and fill in values for
+``settings.GOOGLE_SERVICE_EMAIL``, ``settings.GOOGLE_P12_PATH``, and
+``settings.GOOGLE_PROJECT_ROOT_ID``. Set up and correct values for
+those settings are described in `the Google Apps documentation
+<https://developers.google.com/drive/web/delegation>`_. Pay particular
```

Comment:
this now redirects to https://developers.google.com/drive/api/v3/about-auth#OAuth2Authorizing, and even looking at the section you highlighted, it doesn’t feel too useful (it’s talking about 3-legged oauth and consent screens and stuff). Is there better documentation anywhere?